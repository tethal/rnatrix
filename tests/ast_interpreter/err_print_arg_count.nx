fun main() {
    print(1, 2);
}
