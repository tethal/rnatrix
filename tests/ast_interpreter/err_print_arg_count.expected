<string>:2:5: error: print expects 1 argument
    print(1, 2);
    ^^^^^
